sycl/include/sycl/ext/oneapi/matrix/matrix-tensorcore.hpp

```diff
@@ -495,14 +562,59 @@ struct joint_matrix_mad_impl<
                                      get_layout_pair_id<LayoutA, LayoutB>(), 0);
         }
       }
-    } else if constexpr (std::is_same<T1, double>::value) {
+    } else if constexpr (M == 8 && N == 8 && K == 4) {
```

Comment:
is this change related to bmad addition?

Reply:
No this is a superficial/non-important change that I made just for better consistency of the `if constexpr` statements in this function.